common/djangoapps/third_party_auth/models.py

```diff
@@ -264,6 +262,16 @@ def get_register_form_data(self, pipeline_kwargs):
         # an error on submit.
         registration_form_data['username'] = pipeline_kwargs.get('username')
 
+        # Any other values that are present in the details dict should be copied
+        # into the registration form details. This may include details that do
+        # not map to a value that exists in the registration form. However,
+        # because the fields that are actually rendered are not based on this
+        # list, only those values that map to a valid registration form field
+        # will actually be sent to the form as default values.
+        for blacklisted_field in REGISTRATION_FORM_FIELD_BLACKLIST:
```

Comment:
Nit (feel free to ignore since I don't want to hold this up): Since `REGISTRATION_FORM_FIELD_BLACKLIST` is not used anywhere else, IMHO it is much easier to follow the logic of this code if we just put the value `['name', 'username']` inline here, rather than making the user read something elsewhere in the file to understand what's going on.

Even better, just put `details.pop('name', None)  # Ensure we don't overwrite the name key later` and `details.pop('username', None)` inline above, each on the line following the respective `registration_form_data['...'] = `